core/tests/performance_framework/process_data.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
```

Comment:
I'm wondering if this file should be called ``perf_domain.py`` and the fetcher ``perf_services.py``. Thoughts?